packages/web3-eth-ens/src/ens.ts

```diff
@@ -226,8 +226,8 @@ export class ENS extends Web3Context<EthExecutionAPI & Web3NetAPI> {
 	/*
 	 * Resolves an ENS name to an Ethereum address.
 	 */
-	public async getAddress(ENSName: string) {
-		return this._resolver.getAddress(ENSName);
+	public async getAddress(ENSName: string, coinType = 60) {
```

Comment:
type of `coinType` should also be added here and in other files as well where its without type.